hazelcast-client-legacy/src/main/java/com/hazelcast/client/proxy/ClientRingbufferProxy.java

```diff
@@ -103,7 +102,7 @@ public long remainingCapacity() {
     public long add(E item) {
         checkNotNull(item, "item can't be null");
 
-        AddRequest request = new AddRequest(getName(), toData(item));
+        AddRequest request = new AddRequest(getName(), toData(item), OverflowPolicy.OVERWRITE);
```

Comment:
Interesting, then we new client and old client is not matching . 
@pveentjer can you take a look ? Which OverflowPolicy should we use in sync add method ?